tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -622,8 +890,405 @@ public struct Monster: FlatBufferObject {
+		Monster.add(color: obj.color, &builder)
+		if let o = obj.test?.type {
+			Monster.add(testType: o, &builder)
+			Monster.add(test: __test, &builder)
+		}
+		Monster.addVectorOf(test4: __test4, &builder)
+		Monster.addVectorOf(testarrayofstring: __testarrayofstring, &builder)
+		Monster.addVectorOf(testarrayoftables: __testarrayoftables, &builder)
+		Monster.add(enemy: __enemy, &builder)
+		Monster.addVectorOf(testnestedflatbuffer: __testnestedflatbuffer, &builder)
+		Monster.add(testempty: __testempty, &builder)
+		Monster.add(testbool: obj.testbool, &builder)
+		Monster.add(testhashs32Fnv1: obj.testhashs32Fnv1, &builder)
+		Monster.add(testhashu32Fnv1: obj.testhashu32Fnv1, &builder)
+		Monster.add(testhashs64Fnv1: obj.testhashs64Fnv1, &builder)
+		Monster.add(testhashu64Fnv1: obj.testhashu64Fnv1, &builder)
+		Monster.add(testhashs32Fnv1a: obj.testhashs32Fnv1a, &builder)
+		Monster.add(testhashu32Fnv1a: obj.testhashu32Fnv1a, &builder)
+		Monster.add(testhashs64Fnv1a: obj.testhashs64Fnv1a, &builder)
+		Monster.add(testhashu64Fnv1a: obj.testhashu64Fnv1a, &builder)
+		Monster.addVectorOf(testarrayofbools: __testarrayofbools, &builder)
+		Monster.add(testf: obj.testf, &builder)
+		Monster.add(testf2: obj.testf2, &builder)
+		Monster.add(testf3: obj.testf3, &builder)
+		Monster.addVectorOf(testarrayofstring2: __testarrayofstring2, &builder)
+		Monster.addVectorOf(testarrayofsortedstruct: __testarrayofsortedstruct, &builder)
+		Monster.addVectorOf(flex: __flex, &builder)
+		Monster.addVectorOf(test5: __test5, &builder)
+		Monster.addVectorOf(vectorOfLongs: __vectorOfLongs, &builder)
+		Monster.addVectorOf(vectorOfDoubles: __vectorOfDoubles, &builder)
+		Monster.add(parentNamespaceTest: __parentNamespaceTest, &builder)
+		Monster.addVectorOf(vectorOfReferrables: __vectorOfReferrables, &builder)
+		Monster.add(singleWeakReference: obj.singleWeakReference, &builder)
+		Monster.addVectorOf(vectorOfWeakReferences: __vectorOfWeakReferences, &builder)
+		Monster.addVectorOf(vectorOfStrongReferrables: __vectorOfStrongReferrables, &builder)
+		Monster.add(coOwningReference: obj.coOwningReference, &builder)
+		Monster.addVectorOf(vectorOfCoOwningReferences: __vectorOfCoOwningReferences, &builder)
+		Monster.add(nonOwningReference: obj.nonOwningReference, &builder)
+		Monster.addVectorOf(vectorOfNonOwningReferences: __vectorOfNonOwningReferences, &builder)
+		if let o = obj.anyUnique?.type {
+			Monster.add(anyUniqueType: o, &builder)
+			Monster.add(anyUnique: __anyUnique, &builder)
+		}
+		if let o = obj.anyAmbiguous?.type {
+			Monster.add(anyAmbiguousType: o, &builder)
+			Monster.add(anyAmbiguous: __anyAmbiguous, &builder)
+		}
+		Monster.addVectorOf(vectorOfEnums: __vectorOfEnums, &builder)
+		Monster.add(signedEnum: obj.signedEnum, &builder)
+
+		return Monster.endMonster(&builder, start: __root)
+	}
 }
 
+public class MonsterT: FlatBuffersObjectAPI {
+
+	var pos: MyGame.Example.Vec3T?
+	var mana: Int16
+	var hp: Int16
+	var name: String?
```

Comment:
The name is `required` so you could consider leaving out the `?` ?

Reply:
we simply have it with `?` just because the Buffer would return it optional (to be safe all the time)